src/webgpu/api/operation/vertex_state/correctness.spec.ts

```diff
@@ -562,15 +562,21 @@ struct VSOutputs {
       }
     }
 
-    const expectedDataBG = this.device.createBindGroup({
+    return this.device.createBindGroup({
       layout: pipeline.getBindGroupLayout(0),
       entries: bgEntries,
     });
+  }
 
+  createVertexBuffers(
+    state: VertexLayoutState<{ vbOffset?: number }, TestData>,
+    vertexCount: number = 20,
+    instanceCount: number = 20
```

Comment:
These two optional arguments are never passed. I assume they should be non-optional and passed through from runTest? [1]